### PR TITLE
Update macs2.py

### PR DESCRIPTION
**Pull request by avilella**
fragments|tags for compatibility with older versions of MACS2.

### CODE REVIEW

File: multiqc/modules/macs2/macs2.py [hunk 1]
```diff
@@ -44,13 +44,13 @@ def parse_macs(self, f):
         regexes = {
             'name': r"# name = (.+)$",
             'fragment_size': r"# fragment size is determined as (\d+) bps",
-            'treatment_fragments_total': r"# total fragments in treatment: (\d+)",
-            'treatment_fragments_after_filtering': r"# fragments after filtering in treatment: (\d+)",
-            'treatment_max_duplicates': r"# maximum duplicate fragments in treatment = (\d+)",
+            'treatment_fragments_total': r"# total fragments|tags in treatment: (\d+)",
```
Comment (ewels) [inline multiqc/modules/macs2/macs2.py]: Does this work? Just looking at the code I don't think it should... I'd imagine that you need a non-capturing regex group match:

```python
'treatment_fragments_total': r"# total (?:fragments|tags) in treatment: (\d+)"
```

https://regex101.com is useful for working with these.

Comment (ewels) [reply multiqc/modules/macs2/macs2.py]: Strange! The [automated tests](https://travis-ci.org/ewels/MultiQC/builds/309530045) all agree with me however... See error [here](https://travis-ci.org/ewels/MultiQC/jobs/309530053#L2662-L2675):

```python
[ERROR  ]         multiqc : Oops! The 'macs2' MultiQC module broke... 
  Please copy the following traceback and report it at https://github.com/ewels/MultiQC/issues 
  If possible, please include a log file that triggers the error - the last file found was:
    data/modules/macs2/macs2_output_peaks.xls
============================================================
Module macs2 raised an exception: Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.13/lib/python2.7/site-packages/multiqc-1.4.dev0-py2.7.egg/EGG-INFO/scripts/multiqc", line 412, in multiqc
    output = mod()
  File "/home/travis/virtualenv/python2.7.13/lib/python2.7/site-packages/multiqc-1.4.dev0-py2.7.egg/multiqc/modules/macs2/macs2.py", line 29, in __init__
    self.parse_macs(f)
  File "/home/travis/virtualenv/python2.7.13/lib/python2.7/site-packages/multiqc-1.4.dev0-py2.7.egg/multiqc/modules/macs2/macs2.py", line 66, in parse_macs
    parsed_data[k] = float(match.group(1).strip())
AttributeError: 'NoneType' object has no attribute 'strip'
============================================================
```

Comment (avilella) [reply multiqc/modules/macs2/macs2.py]: Feel free to change it to the way it should be.

Comment (ewels) [reply multiqc/modules/macs2/macs2.py]: It's quite a lot of hassle for me to do this - I have to create a new repository branch, edit your PR to point to that, pull your branch, make the update, then merge that into the repo.

By contrast, if you can make the change based on what I pasted above then it's just a `git push` for you and I can merge cleanly. So I'd prefer that please :)

Comment (avilella) [reply multiqc/modules/macs2/macs2.py]: Sure.

Comment (ewels) [reply multiqc/modules/macs2/macs2.py]: Great, thanks! No need to close the pull-request - just push a new commit and it will update..

Comment (ewels) [reply multiqc/modules/macs2/macs2.py]: Ah you did it from a patch branch this time, ok..

**Pull request closed without merging**